### DISCUSSION TITLE
Features to avoid in IE11 while still using Tailwind 2.0+

### DISCUSSION THREAD

**Discussion by jlkiri** [tailwindlabs/tailwindcss#3466] [Help] [answered]
I know Tailwind 2 dropped IE11 support, but since most classes still should be compatible, is there a list of features I can manually avoid to be able to use Tailwind 2 in IE11? Like `grid` etc.

**Comment by adamwathan** [comment, marked as answer]
Even basic classes like `bg-black` won't work because they use CSS variables under the hood. If you need IE11 support best to stick with v1.9 👍

**Reply by jlkiri** [reply]
I see, thanks!

**Reply by ccchapman** [reply]
Thanks for this answer, it is helpful! I am starting a new project with _basic_ IE 11 support and was also curious the extent at which it was not supported. It sounds like there could  theoretically be community support in the future, per the [discussion here](https://github.com/tailwindlabs/tailwindcss/discussions/2652), but we should definitely just stick with v1.9 for now to be safe.

**Reply by Marcelx8** [reply]
Thanks for this forum topic. I have to cater for IE11 also in some cases. The only issue I have so far is the variables. After the struggle, I started with v1.9 - well I think it was 1.9. Package.json for my Vuejs app stated 1.9.6, but it seems that Vue requires a compatibility version that includes Postcss. I can't seem to find a compatible version for 1.9 to cover my needs of IE11. Can anyone point me in a direction? I'll start a small project to test, without Vue. I just need css that works with ie11.

**Reply by vpress-admin** [reply]
Are there any guides on how to downgrade from 2.x to 1.9? I've built my whole app using 2.x but have been informed last minute that IE11 support is required :'(
